net-mail/dot-forward/dot-forward-0.71-r5.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fixheadtails qmail
+
+DESCRIPTION="reads sendmail's .forward files under qmail"
+HOMEPAGE="http://cr.yp.to/dot-forward.html"
+SRC_URI="http://cr.yp.to/software/${P}.tar.gz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm ~hppa ~ia64 ~m68k ~mips ppc ppc64 ~s390 sparc x86"
```

Comment:
You've kept stable keywords. Either git mv in this case or copy and ekeyword ~all.